sol.h

```diff
@@ -195,7 +195,8 @@ struct undo undo_sentinel;
 	LONGHELP_SPECIFIC \
 	"    -b(land colorscheme)\n" \
 	"    -c(olorful colorscheme)\n" \
-	"    -m(iniature colorscheme)\n" \
+	"    -m(iniature colorscheme, bland)\n" \
```

Comment:
call it 'monochrome' instead of bland please. that name was just a hack, so it starts with the correct letter.

Reply:
that applies to the getopt part as well